testsuite/tests/lib-format/print_seq.ml

```diff
@@ -0,0 +1,34 @@
+(* TEST
+   include testing
+*)
+
+(*
+
+A test file for the Format module.
+
+*)
+
+open Testing;;
+open Format;;
+
+let say s = Printf.printf s;;
+
+let pp_print_intseq = pp_print_seq ~pp_sep:pp_print_space pp_print_int
+
+try
```

Comment:
This is not valid syntax. You can preface the expression with `let () = try ...` instead.

Reply:
I use the style from another test in the same directory, and it uses `;;` termination rather than `let () =` at top level. I made a mistake when adding this intermediate function.

It's fixed now. I also added a corresponding `reference` file and I ran the test locally.